include/gotcha/gotcha_types.h

```diff
@@ -28,15 +28,17 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 extern "C" {
 #endif
 
+typedef void* gotcha_wrappee_handle_t;
+
 /*!
  * The representation of a Gotcha action
  * as it passes through the pipeline
  */
 struct gotcha_binding_t {
-  const char* name;                //!< The name of the function being wrapped
-  void* wrapper_pointer;           //!< A pointer to the wrapper function
-  void* function_address_pointer;  //!< A pointer to the function being wrapped
-  void* opaque_handle;             //!< This pointer is for Gotcha developer use only
+  const char* name;                                //!< The name of the function being wrapped
+  void* wrapper_pointer;                           //!< A pointer to the wrapper function
+  gotcha_wrappee_handle_t function_address_pointer;  //!< A pointer to the function being wrapped
```

Comment:
Change comment from "a pointer to" to "a handle for".  Also change variable name to "function_handle".